makefile

```diff
@@ -46,8 +46,10 @@ test_all:
 	  @echo 'Core tests'    >> build/test_summary.txt
 	  make core_tests || (echo oops SMT failed | tee -a build/test_summary.txt)
 	  @echo ''              >> build/test_summary.txt
-	  @echo 'Serpent tests' >> build/test_summary.txt
-	  make serpent_tests || (echo oops serpent failed | tee -a build/test_summary.txt)
+	  # @echo 'Serpent tests' >> build/test_summary.txt
+	  # make serpent_tests || (echo oops serpent failed | tee -a build/test_summary.txt)
```

Comment:
Also, I vote for not including commented code like this. Either remove it (it can be found in the git commit history if we need it) or move it to some other make rule for running the serpent flow.